src/components/input-group/utils/inputFielfGroup.enum.ts

```diff
@@ -0,0 +1,26 @@
+export enum EInputFieldGroupType {
```

Comment:
is there a benefit of defining all possible values in an enum or is it just personal preference?

Reply:
Well, Personal preference. Also wanted to keep the consistency within all components. 
I find it more elegant even if it produces a bit more code.